src/helpers/lineHeight.js

```diff
@@ -0,0 +1,19 @@
+import { css } from 'styled-components'
```

Comment:
Could be good to add an entry example in the helpers section of the styleguidist ?